contracts/DxMgnPool.sol

```diff
@@ -138,7 +140,7 @@ contract DxMgnPool is Ownable {
         // Don't revert if wen can't claimSellerFunds
         address(dx).call(abi.encodeWithSignature("claimSellerFunds(address,address,address,uint256)", secondaryToken, depositToken, address(this), lastParticipatedAuctionIndex));
         mgnToken.unlockTokens();
-        totalDeposit = dx.balances(address(depositToken), address(this));
+        totalDeposit = dx.balances(address(depositToken), address(this)) + depositToken.balanceOf(address(this));
```

Comment:
This fix should probably be a separate PR. Also, did you check if `dx.withdraw` reverts if we try to withdraw more than the balance inside `dx`? Maybe it'd be safer to only withdraw `dx.balances(address(depositToken), address(this))`